src/toil/options/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from toil.lib.conversions import strtobool
+
+
+def convert_bool(b: str) -> bool:
+    """Convert a string representation of bool to bool"""
+    return bool(strtobool(b))
```

Comment:
This function just does exactly what `strtobool` does. We probably should get rid of it and just use `strtobool`.